packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -2,108 +2,103 @@
 pragma solidity 0.8.15;
 
 import { L1Block } from "src/L2/L1Block.sol";
+import { EnumerableSetLib } from "@solady/utils/EnumerableSetLib.sol";
```

Comment:
Any update on the analysis between using solady and oz?

Reply:
went for oz bc it doesn't have issue of `_ZERO_SENTINEL`